mypy/test/testcheck.py

```diff
@@ -281,8 +295,9 @@ def parse_module(self, program_text: str, incremental: int = 0) -> List[Tuple[st
         Return a list of tuples (module name, file name, program text).
         """
         m = re.search('# cmd: mypy -m ([a-zA-Z0-9_. ]+)$', program_text, flags=re.MULTILINE)
-        m2 = re.search('# cmd2: mypy -m ([a-zA-Z0-9_. ]+)$', program_text, flags=re.MULTILINE)
-        if m2 is not None and incremental == 2:
+        m2 = re.search('# cmd{}: mypy -m ([a-zA-Z0-9_. ]+)$'.format(incremental_step),
```

Comment:
I think that this works correctly since we include the incremental step number in the regexp, so this lets you use `# cmd2` and `# cmd3` together, since the function is being called once per incremental step. The name `m2` is kind of confusing though. I'll refactor this code a bit to make it clearer. `# cmdlol` is not accepted.